docs/chapters/workflow/release/release-process.rst

```diff
@@ -108,7 +134,7 @@ Release notes
     ``PELUX_<VERSION>_Release_Notes`` where <VERSION> is the version number of
```

Comment:
Binary images for the supported variants in .wic format

Reply:
WIC is Intel's format